admin/class-automation.php

```diff
@@ -43,6 +43,46 @@ class Automation {
 			'location' => 'article_metadata',
 			'type'     => 'boolean',
 		],
+		'Section'     => [
+			'location' => 'apple_news_api',
+			'type'     => 'string',
+		],
+		'Slug'        => [
+			'location' => 'user_input',
+			'type'     => 'string',
+		],
+		'Theme'       => [
+			'location' => 'apple_new_api',
```

Comment:
Set this to `exporter`
```suggestion
			'location' => 'exporter',
```